### PR TITLE
Handle wallet account change

### PR DESCRIPTION
**Pull request by sebastinez**
Hello @xla and @cloudhead!

I created this PR to handle the eventual wallet account change by the user.
Please let me know any edits you need or improvements.

Cheers!

### CODE REVIEW

File: src/session.ts [hunk 1]
```diff
@@ -154,6 +164,9 @@ export const session = derived(state, s => {
   return null;
 });
 
+// Updates state when user changes accounts
+window.ethereum?.on("accountsChanged", state.setChangedAccount)
```
Comment (cloudhead) [inline src/session.ts:168]:
```suggestion
window.ethereum?.on("accountsChanged", state.setChangedAccount);
```

**Comment by cloudhead** [comment]
Very nice! Have you confirmed locally that it works as expected?

**Comment by sebastinez** [comment]
> Very nice! Have you confirmed locally that it works as expected?

Yes, I ran the build in Firefox and Chrome with Metamask and did use different scenarios:
- with Metamask disabled
- with Metamask disabled but enabling it with the page open and refreshing, to have the `window.ethereum` provider.
- changing accounts inside Metamask

**Comment by cloudhead** [comment]
Awesome, thanks!

**Review by cloudhead — APPROVED** [review]

**Pull request merged**